Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Buttons/Controls Toolkit/Command Links/KryptonCommandLinkVersion2.cs

```diff
@@ -40,7 +40,7 @@ public bool UseAsUACElevatedButton
             {
                 _useAsUACElevatedButton = value;
 
-                // TODO: Store the original icon
+                // Note: Store the original icon
```

Comment:
This is still a todo: in order to restore back when UAC is off !

Reply:
Done